### ISSUE TITLE
Ambiguity in docs about ORPCA and RPCA

### ISSUE THREAD

**Issue by thomasaarholt** [hyperspy/hyperspy#2598]
From the docs:
> Finally, online RPCA includes two alternatives methods

But then the code examples use RPCA, and not ORPCA:

```python
> s.decomposition(algorithm="RPCA",
>                 output_dimension=3,
>                 method="SGD",
>                 subspace_learning_rate=1.1)
```
Do the SGD methods apply to RPCA as well? Or are these just being passed into the function and do nothing?

A second question: Does anyone have a good approach for estimating the output_dimension on big data, where the number of components is somewhere between 3 and 20? How should I perform decomposition on such data? Start with output_dimension=20, see how many components are noise, and then reduce by that number?

**Comment by tjof2** [comment]
@thomasaarholt 

1) is a bug in the docs - SGD does not apply to RPCA. Code example should be `algorithm="ORPCA"`.

2) For big data, start small (3) and increase. Has the same effect as starting at 20 and decreasing, but it is more efficient in terms of speed/memory if your dataset is really large.

**Comment by ericpre** [comment]
Doc fixed in #2662.
